config/routes.rb

```diff
@@ -15,7 +15,7 @@
     resources :holds
   end
   resources :books
-  resources :schools, :only => [:index]
+  resources :schools, :only => [:index, :create]
```

Comment:
Are we adding functionality to create schools?

Reply:
@daryanypl  this method created long back.   I am writing test cases.
  def create
    School.create(school_params)
  end

  private

  # Strong parameters: protect object creation and allow mass assignment.
  def school_params
    params.permit(:name, :code, :active, :address_line_1, :address_line_2, :state, :postal_code, :phone_number, :borough)
  end